docs/application/native/guides/device/system.md

```diff
@@ -409,7 +409,11 @@ The following table lists the network feature keys:
 | `http://tizen.org/feature/network.bluetooth.hid.device` | `bool` | The platform returns `true` for this key, if the device supports the Bluetooth Human Interface Device (HID) device feature. |
 | `http://tizen.org/feature/network.bluetooth.le` | `bool` | The platform returns `true` for this key, if the device supports Bluetooth Low Energy (BLE). |
 | `http://tizen.org/feature/network.bluetooth.le.5_0` | `bool` | The platform returns `true` for this key, if the device supports Bluetooth 5 feature. |
+| `http://tizen.org/feature/network.bluetooth.le.gatt.client` | `bool` | The platform returns `true` for this key, if the device supports the Bluetooth Generic Attribute Profile (GATT) Client. |
+| `http://tizen.org/feature/network.bluetooth.le.gatt.server` | `bool` | The platform returns `true` for this key, if the device supports the Bluetooth Generic Attribute Profile (GATT) Server. |
+| `http://tizen.org/feature/network.bluetooth.le.ipsp` | `bool` | The platform returns `true` for this key, if the device supports Bluetooth Internet Protocol Support Profile (IPSP). |
 | `http://tizen.org/feature/network.bluetooth.opp` | `bool` | The platform returns `true` for this key, if the device supports Bluetooth Object Push Profile (OPP). |
```

Comment:
Question to this file and tutorials/details/app-filtering.md.

Shouldn't `http://tizen.org/feature/network.bluetooth.oob` come before `http://tizen.org/feature/network.bluetooth.opp`?

Reply:
done